src/screens/Blocked/elements/DomainScreen/index.js

```diff
@@ -54,7 +54,7 @@ const BlockedDomainList = (props) => {
       keyExtractor={(item, index) => index.toString()}
       refreshing={isLoading}
       onRefresh={handleFetchData}
-      style={{backgroundColor: COLORS.white}}
+      style={{backgroundColor: COLORS.almostBlack}}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.white` to `COLORS.almostBlack` alters the background color of the component. Ensure this color change aligns with the overall design scheme and is accessible (i.e., provides sufficient contrast) for all users.
```diff
- style={{backgroundColor: COLORS.white}}
+ style={{backgroundColor: COLORS.almostBlack}}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->